### PR TITLE
New package: DanielMcGuireCorporation.SimpleBrowser version 2.3.0

### PR DESCRIPTION
**Pull request by DanielLMcGuire**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138267)

### CODE REVIEW

**Comment by DanielLMcGuire** [comment]
@microsoft-github-policy-service agree [company="{Daniel McGuire Corporation}"]
I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
@microsoft-github-policy-service agree
 am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
 @microsoft-github-policy-service agree company="Microsoft"
Contributor License Agreement

**Comment by DanielLMcGuire** [comment]
@microsoft-github-policy-service agree

**Comment by DanielLMcGuire** [comment]
@microsoft-github-policy-service agree

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
This manifest contains a term that is blocked:

> License: DMC EULA

(Automated response - build 720.)

**Comment by stephengillie** [comment]
> This manifest contains a term that is blocked:
> 
> > License: DMC EULA
> 
> (Automated response - build 720.)

Oh - in this case, my automation was in error. It's meant to catch license installer switches, but checks all files. (I need to improve its filtering.) 

Feel free to revert your license. (If this is a valid license type - I'm not a licensing lawyer.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-09T14:28:42.9564886Z ##[error] One or more errors occurred. (SQL error or missing database (0x87AF0001)) SQL error or missing database (0x87AF0001)

(Automated response - build 725.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

This appears to be `Microsoft.DotNet.DesktopRuntime.8`.

File: manifests/d/DanielMcGuireCorporation/SimpleBrowser/2.3.0/DanielMcGuireCorporation.SimpleBrowser.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: DanielMcGuireCorporation.SimpleBrowser
+PackageVersion: 2.3.0
+InstallerType: nullsoft
```
Comment (stephengillie) [inline manifests/d/DanielMcGuireCorporation/SimpleBrowser/2.3.0/DanielMcGuireCorporation.SimpleBrowser.installer.yaml:6]:
```suggestion
InstallerType: nullsoft
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**